docs/user/04-metrics.md

```diff
@@ -299,14 +299,14 @@ You only need to have the annotations in place for metrics ingestion to start au
 
 Put the following annotations either to a Service that resolves your metrics port, or directly to the Pod:
 
-```yaml
-prometheus.io/scrape: "true"   # mandatory to enable automatic scraping
-prometheus.io/scheme: https    # optional, default is "http" if no Istio sidecar is used. When using a sidecar (Pod has label `security.istio.io/tlsMode=istio`), the default is "https". Use "https" to scrape workloads using Istio client certificates.
-prometheus.io/port: "1234"     # optional, configure the port under which the metrics are exposed
-prometheus.io/path: /myMetrics # optional, configure the path under which the metrics are exposed
-```
+| Annotation Key         | Example Values    | Default Value | Description                                                                                                                                                                                                                                                                                                                                 |
+|------------------------|-------------------|--------------- |---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `prometheus.io/scrape` | `true`, `false` | None (mandatory) | Controls whether Prometheus will automatically scrape metrics from this target.                                                                                                                                                                                                                                                             |
```

Comment:
I was a bit bamboozled by the "None (mandatory)", because my first thought was that it's mandatory to have **_no_** value. 
I wonder if we should put the "mandatory" remark either into the first column (after the key) or into the "Description" column:

```suggestion
| `prometheus.io/scrape` (mandatory) | `true`, `false` | None | Controls whether Prometheus will automatically scrape metrics from this target.                                                                                                                                                                                                                                                             |
```
or 

```suggestion
| `prometheus.io/scrape` | `true`, `false` | None | Mandatory. Controls whether Prometheus will automatically scrape metrics from this target.                                                                                                                                                                                                                                                             |
```